### PR TITLE
Fix canary tests on CIs (travis, appveyor)

### PR DESCRIPTION
**Pull request by arunoda**
Basically this increases the Jest default timeout to 5 min.
So, it has plenty of time to build the app.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "testonly": "cross-env NODE_PATH=test/lib jest \\.test.js",
     "posttestonly": "taskr posttest",
     "pretest": "npm run lint",
-    "test": "npm run testonly -- --coverage --forceExit --runInBand --verbose --bail",
+    "test": "npm run testonly -- --coverage --forceExit --runInBand --verbose --bail --testPathPattern production",
```
Comment (timneutkens) [inline package.json]: Should this be only production? 🤔

File: test/integration/production/test/index.test.js [hunk 2]
```diff
@@ -37,7 +37,8 @@ describe('Production Usage', () => {
   afterAll(() => stopApp(server))
 
   describe('With basic usage', () => {
-    it('should render the page', async () => {
+    it('should render the page 11', async () => {
+      // throw new Error(`ERRRO-${context.appPort}`)
```
Comment (timneutkens) [inline test/integration/production/test/index.test.js]: Looks like an artifact from testing too

Comment (arunoda) [reply test/integration/production/test/index.test.js]: yes.

**Comment by comus** [comment]
@arunoda 
just a related issue
after `npm run test` completed. and i see those `next-dev` not kill
just `npm run test`
and `ps -ax | grep next`
will see
```
16276 ttys000    0:55.11 node /Users/chris/Desktop/next.js/dist/bin/next-dev /Users/chris/Desktop/next.js/test/integration/basic/ -p 20615
16980 ttys000    0:49.89 node /Users/chris/Desktop/next.js/dist/bin/next-dev /Users/chris/Desktop/next.js/test/integration/basic/ -p 27159
17575 ttys000    0:10.53 node /Users/chris/Desktop/next.js/dist/bin/next-dev /Users/chris/Desktop/next.js/test/integration/ondemand/ -p 26044
```

**Comment by arunoda** [comment]
@comus 
hmm. Not related to this.
May be related to some other issue.

**Pull request merged**